components/Layout/Footer/Footer.module.scss

```diff
@@ -0,0 +1,51 @@
+.footer {
+    padding: 3rem 5rem 1rem 0;
+    position: relative;
+    line-height: 1.5;
+    width: 100%;
+    overflow: hidden;
+
+    @include media-query(medium) {
+        align-self: flex-end;
+        display: flex;
+        align-items: center;
+        width: auto;
+    }
+
+    .copyright {
+        margin-right: 0.25rem;
+
+        @include media-query(medium) {
+            font-weight: $weight--bold;
+        }
```

Comment:
```suggestion
```
We can get rid of this extra style rule as `.copyright` is in a strong tag.

Reply:
Correction: maybe we can keep this in, but replace the strong tag with a span instead (see below)

Reply:
See comment below